docs/deployment/containers/docker.txt

```diff
@@ -192,26 +208,55 @@ Read about Docker Compose specifics `here <https://docs.docker.com/compose/>`_.
 
 .. code-block:: yaml
 
-    crate-1:
+    crate01:
+      image: crate
+      container_name: crate01
+      hostname: crate01
+      ports:
+        - 4201:4200
+      net: crate
+      volumes:
+        - /tmp/crate/01:/data
+      command: >
+        crate -Cnetwork.host=_site_
```

Comment:
also this node should have its seed_hosts and initial_master_nodes configured

Reply:
Pointing to itself?, or the other two nodes?. Should each node be aware of the other two?

Reply:
yes each node should be aware of all other nodes as the order in how they are started may not be deterministic and the configuration should not rely on that